src/service/resource.js

```diff
@@ -227,6 +227,9 @@ export class Resource {
     /** @const @private {boolean} */
     this.intersect_ = resources.isIntersectionExperimentOn();
 
+    /** @const @private {boolean} */
+    this.isInViewport_ = false;
```

Comment:
Please add `TODO(#out-tracking-bug)` to remove with `whenInViewport`

Reply:
Sure! I'll add these two more final items in the remove viewportCallback issue.

Reply:
done